lib/messages.js

```diff
@@ -3,6 +3,34 @@ var _ = require('underscore'),
     utils = require('./utils'),
     logger = require('./logger');
 
+function extendedTemplateContext(doc, extras) {
```

Comment:
You mean arrow function? I'm following the convention in the rest of the file. Some day it will be converted to ES6, but not today!